samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/bundle/EvaluationMessages_eu.properties

```diff
@@ -361,9 +361,9 @@ export_as_pdf=Esportatu PDFa
 export_as_xlsx=Esportatu Excela
 export_pdf=PDF gisa
 export_xlsx=Excel gisa
+export_site=Site
```

Comment:
@pablosanroman Site sería Gune en Euskera? Quizá no..... gracias!

Reply:
Yes, @mpellicer, you are right, and depending on the context, it would have to go with the article: "a", as it goes here:
export_site=Gunea

Reply:
@PabloSanRoman Thanks a lot!